onnxruntime/test/framework/inference_session_test.cc

```diff
@@ -1399,5 +1400,85 @@ TEST(InferenceSessionTests, TestParallelExecutionWithCudaProvider) {
 
 #endif
 
+TEST(InferenceSessionTests, GRUBatchSeqLenIntTest) {
+  static const std::string GRU_MODEL_URI = "testdata/gru_batch_seq_len_imatmul.pb";
+  // This model is a GRU with batch and sequence length, with quantized MatMulInteger
+
```

Comment:
This doesn't seem relevant to testing InferenceSession. Is there a better location for this test?

Reply:
This tests truncated sequence inference with sequence length input, and there's another test of TestTruncatedSequence in this file already. I think it does not belong to provider test, so that's why it landed here. Any other suggestions?

---
In reply to: [312422075](https://github.com/microsoft/onnxruntime/pull/881#discussion_r312422075) [](ancestors = 312422075)

Reply:
Maybe re-name the test if it's about testing support for truncated sequence. Based on the current name it sounds like the main purpose is to test the GRU op.

---
In reply to: [312632050](https://github.com/microsoft/onnxruntime/pull/881#discussion_r312632050) [](ancestors = 312632050,312422075)

Reply:
Good point, will do

---
In reply to: [312761916](https://github.com/microsoft/onnxruntime/pull/881#discussion_r312761916) [](ancestors = 312761916,312632050,312422075)